README.adoc

```diff
@@ -139,7 +139,8 @@ $ curl http://localhost:8080/people
 There are currently no elements and hence no pages. Time to create a new `Person`!
 
 ```
-$ curl -i -X POST -H "Content-Type:application/json" -d '{"firstName": "Frodo", "lastName": "Baggins"}' http://localhost:8080/people
+$ curl -i -X POST -H "Content-Type:application/json" -d "{  \"firstName\" : \"Frodo\",  \"lastName\" : \"Baggins\" }" http://localhost:8080/people
```

Comment:
I prefer the original form. If you had a problem what are the steps to reproduce? Is it an OS-specific issue?

BTW the `-X POST` is redundant (unrelated).